modules/libcom/src/osi/os/default/osdAssert.c

```diff
@@ -50,6 +52,12 @@ void epicsAssert (const char *pFile, const unsigned line,
     errlogPrintf("Please E-mail this message to %s or to tech-talk@aps.anl.gov\n",
         pAuthorName);
 
-    errlogPrintf("Calling epicsThreadSuspendSelf()\n");
-    epicsThreadSuspendSelf ();
+    if (envGetBoolConfigParam(&EPICS_ABORT_ON_ASSERT, &shouldAbort) == 0 && shouldAbort) {
+        errlogPrintf("Calling abort()\n");
```

Comment:
Please use `printf(stderr, ...)` here, the process is about to die and errlog might not wake up in time to do anything.

Reply:
Do you want the rest of the `errlogPrintf`'s changed in this function? It might be better to use `errlogFlush()` before `abort()`.